samtranslator/model/dynamodb.py

```diff
@@ -13,8 +13,9 @@ class DynamoDBTable(Resource):
             'ProvisionedThroughput': PropertyType(True, dict_of(is_str(), one_of(is_type(int), is_type(dict)))),
             'StreamSpecification': PropertyType(False, is_type(dict)),
             'TableName': PropertyType(False, one_of(is_str(), is_type(dict))),
-            'Tags': PropertyType(False, list_of(is_type(dict)))
-        }
+            'Tags': PropertyType(False, list_of(is_type(dict))),
+            'SSESpecification': PropertyType(False, list_of(is_type(dict)))
```

Comment:
Whoops.  I wonder how this ever worked.

Reply:
Ya. I don't think we have good testing around these, mainly because they are the CloudFormation Resources as objects. :(

The line is required but they second param to `PropertyType` should be `is_type(dict)`.

Background, there python magic happening to produce the actually classes. All the vanilla CloudFormation Resources, inherit from `Resource` and in the `__init__` there is this [magical code that creates the properties in the class](https://github.com/awslabs/serverless-application-model/blob/develop/samtranslator/model/__init__.py#L66). I find this confusing and took me a while to understand but it is what it is.